docs/module/lag.md

```diff
@@ -83,17 +88,15 @@ groups:
   device: dellos10
  hosts:
   members: [h1,h2]
-  device: frr        # 'linux' does not support the lag module yet
+  device: frr          # 'linux' does not support the lag module yet
```

Comment:
If the hosts don't support the LAG module, create an access switch to two core switches example.

Reply:
I'd prefer to include https://github.com/ipspace/netlab/pull/1624 and change the example to use Linux hosts